.env.example

```diff
@@ -1,17 +1,17 @@
 # Kafka
-KAFKA_BROKERS=
+export KAFKA_BROKERS=
 
 # Kafka SASL Authentication
-SASL_USERNAME=
-SASL_PASSWORD=
-SASL_MECHANISM=
+export SASL_USERNAME=
+export SASL_PASSWORD=
+export SASL_MECHANISM=
 
```

Comment:
I like the idea of having `export` in the README.md file, but not sure if this is a good practice for the `.env.example`.

In general, with `.env` file in Node.js and other frameworks, you should not use the export keyword within the file itself. 

Not sure if the `dotenv` library, expects the `export` keyword on each line. It usually is just a key-value pair.

Reply:
Dotenv is totally cool with export. It makes it so you can `source /path/to/.env; datagen <blah>`

Reply:
oh ok, in this case we should be good to go!